streaming/text/convert/enwiki_text.py

```diff
@@ -24,72 +24,73 @@ def parse_args() -> Namespace:
         '--in_root',
         type=str,
         required=True,
-        help='Directory path to store the input dataset',
+        help='Local directory path of the input raw dataset',
     )
     args.add_argument(
         '--out_root',
         type=str,
         required=True,
-        help='Directory path to store the output dataset',
+        help='Directory path to store the output MDS shard files',
     )
     args.add_argument(
         '--compression',
         type=str,
         default='zstd:7',
-        help='Compression algorithm to use. Default: zstd:7',
+        help='Compression algorithm to use. Defaults to ``zstd:7``',
     )
     args.add_argument(
         '--hashes',
         type=str,
         default='sha1,xxh64',
-        help='Hashing algorithms to apply to shard files. Default: sha1,xxh64',
+        help='Hashing algorithms to apply to shard files. Defaults to ``sha1,xxh64``',
     )
     args.add_argument(
         '--size_limit',
         type=int,
         default=1 << 25,
-        help='Shard size limit, after which point to start a new shard. Default: 1 << 25',
+        help='Shard size limit, after which point to start a new shard. Defaults to ``1 << 25``',
     )
     args.add_argument(
-        '--progbar',
+        '--progress_bar',
         type=int,
         default=1,
-        help='tqdm progress bar. Default: 1 (True)',
+        help='tqdm progress bar. Defaults to ``1 (True)``',
     )
     args.add_argument(
         '--leave',
         type=int,
         default=0,
-        help='Keeps all traces of the progressbar upon termination of iteration. Default: 0 ' +
-        '(False)',
+        help='Keeps all traces of the progressbar upon termination of iteration. ' +
+        'Defaults to ``0 (False)``',
     )
     return args.parse_args()
 
 
 def process_split(in_root: str, out_root: str, compression: str, hashes: List[str],
-                  size_limit: int, progbar: int, leave: int, basenames: List[str],
+                  size_limit: int, progress_bar: int, leave: int, basenames: List[str],
                   split: str) -> None:
     """Process a dataset split.
 
     Args:
-        in_root (str): Input dataset path.
-        out_root (str): Output dataset path.
+        in_root (str): Local directory path of the input raw dataset.
+        out_root (str): Directory path to store the output MDS shard files.
```

Comment:
technically a dataset, not just shards

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/171/commits/ad49770b6c7f35b3e143cdf0fc6f5f699f7da917.